content/docs/development/health-rules.md

```diff
@@ -0,0 +1,168 @@
+# Health Rules
+
+Cartographer reports the health of each object stamped on the cluster. If no health rule is specified in the template,
+Cartographer's health behavior will be rudimentary:
+
+- if the object is rejected by the API server (e.g. if you mistakenly try to template out a CinfogMap instead of a
+  ConfigMap) then the workload will report the resource's Healthy condition as "Unknown". The workload will report the
+  resource's Ready condition as "False" for the reason "TemplateRejectedByAPIServer".
+- if the object is created but the path that is meant to be read does not exist (e.g. you have a ClusterImageTemplate
+  where the imagePath mistakenly points to a non-existent field) then the workload will report the resource's Healthy
+  condition as "Unknown" for the reason "OutputNotAvailable". The workload will report the resource's Ready condition as
+  "Unknown" for the reason "MissingValueAtPath".
+
+Otherwise, the object's Healthy condition will be "True". By defining one of several types of health rules, a template
+author can better define the conditions that report healthy or unhealthy status. This will ensure that the proper status
+is returned to users.
+
+## Always Healthy
+
+Some resources simply have no sense of "healthiness" and are meant to be considered always healthy. For instance, a
+ConfigMap would never be unhealthy as it always simply reflects the data values that were last submitted.
+
+Template authors may choose not specify a healthrule at all (i.e., rely on the default of being always healthy). But it
+is clearer to be explicit:
```

Comment:
```suggestion
```
What do you think about removing this, since we've now established rudimentary behavior isn't the same as AlwaysHealthy